startup/31-xspress3.py

```diff
@@ -262,20 +262,21 @@ def unstage(self):
 
 try:
     xs = SrxXspress3Detector("XF:05IDD-ES{Xsp:1}:", name="xs")
-    if "TOUCHBEAMLINE" in os.environ and os.environ["TOUCHBEAMLINE"] == 1:
-        xs.channel1.rois.read_attrs = ["roi{:02}".format(j)
-                                       for j in [1, 2, 3, 4]]
-        xs.channel2.rois.read_attrs = ["roi{:02}".format(j)
-                                       for j in [1, 2, 3, 4]]
-        xs.channel3.rois.read_attrs = ["roi{:02}".format(j)
-                                       for j in [1, 2, 3, 4]]
-        xs.channel4.rois.read_attrs = ["roi{:02}".format(j)
-                                       for j in [1, 2, 3, 4]]
-        xs.hdf5.num_extra_dims.put(0)
+    xs.channel1.rois.read_attrs = ["roi{:02}".format(j)
+                                   for j in [1, 2, 3, 4]]
+    xs.channel2.rois.read_attrs = ["roi{:02}".format(j)
+                                   for j in [1, 2, 3, 4]]
+    xs.channel3.rois.read_attrs = ["roi{:02}".format(j)
+                                   for j in [1, 2, 3, 4]]
+    xs.channel4.rois.read_attrs = ["roi{:02}".format(j)
+                                   for j in [1, 2, 3, 4]]
+    if "TOUCHBEAMLINE" in os.environ and os.environ["TOUCHBEAMLINE"] == '1':
```

Comment:
A bit shorter version:
```suggestion
    if os.getenv("TOUCHBEAMLINE", "0") == "1":
```